test/check-application

```diff
@@ -1334,7 +1334,7 @@ class TestApplication(testlib.MachineCase):
         b.wait_visible('#containers-images td[data-label="Image"]:contains("busybox:latest")')
         b.click('#containers-containers tbody tr:contains("busybox:latest") td.pf-c-table__toggle button')
         b.click('#containers-images tbody tr:contains("busybox:latest") .ct-container-create')
-        b.wait_visible('div.pf-c-modal-box header:contains("Run image")')
+        b.wait_visible('div.pf-c-modal-box header:contains("Create container")')
```

Comment:
Can you also test that new checkbox - that you will unset it and make sure it does not start?

Reply:
Added a test.